packages/protocol/test/common/metatransactionwallet.ts

```diff
@@ -591,4 +599,30 @@ contract('MetaTransactionWallet', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#transferCeloDollarsToSigner', () => {
+    const value = 100
+    let mockStableToken: MockStableTokenInstance
+    beforeEach(async () => {
+      const Registry: RegistryContract = artifacts.require('Registry')
+      const registry: RegistryInstance = await Registry.at(
+        '0x000000000000000000000000000000000000ce10'
+      )
+      // Problem is this is owned by governance...
+      console.log(await registry.owner())
+      console.log(accounts)
+      const MockStableToken: MockStableTokenContract = artifacts.require('MockStableToken')
+      mockStableToken = await MockStableToken.new()
+      await registry.setAddressFor(CeloContractName.StableToken, mockStableToken.address)
+      await mockStableToken.mint(signer, value)
```

Comment:
Why do we mint to `signer`, not `wallet`?

Reply:
Oops